modules/va_notify/README.md

```diff
@@ -111,14 +111,131 @@ Using option #2:
+
+Here is an example:
+```
+if Flipper.enabled?(:custom_callback_handler)
+  VANotify::EmailJob.perform_async(
+    user.va_profile_email,
+    template_id,
+    get_personalization(first_name),
+    Settings.vanotify.services.va_gov.api_key,
+    { callback_klass: 'VANotify::CustomNotificationCallback', callback_metadata: {  statsd_tags: { service: 'ExampleTeam' } } }
+  )
+else
+  # Default logic
+end
+```
+---
+
+#### Behind the Scenes: How Callbacks Work
+
+Here's a high-level overview of what happens behind the scenes when using VA Notify callbacks:
+
+1. Notification Sending: When a notification is sent via the VA Notify API, a notification_id is generated and returned. This ID should be saved to track the delivery status.
+
+2. Delivery Processing: VA Notify attempts to deliver the notification using its internal delivery workflow. This includes retries for temporary issues and contact lookups if an ICN is used.
+
+3. Callback Triggered: As the delivery progresses, VA Notify sends status updates to the configured callback URL. Updates may include statuses like "delivered," "failed," or "temporary failure".
+
+4. Processing Callback: Your application receives the callback and processes it to determine if further action is needed—such as notifying the user of a failed delivery, retrying, or marking the notification as successfully delivered.
+
+5. Monitoring: Metrics and logs capture the outcome for reporting and troubleshooting.
+
+---
+
+#### Delivery Status Reference
+
+Refer to the [VA Notify Error Status Mapping Table](https://github.com/department-of-veterans-affairs/vanotify-team/blob/main/Support/error_status_reason_mapping.md#error-table) for detailed status codes and their meanings.
+
+---
 
-A successful request to VA Notify API does not necessarily mean that your intended recipient will actually receive the notification due to the errors mentioned above. Please set up delivery status callbacks, so you can track the success or failure of your notification in real time. VA Notify provides aggregated metrics, but does not provide historical Veteran level metrics.
+#### Contact Us
 
-To track the delivery status of individual notifications you will need to set up service callbacks. These callbacks will allow you to determine the delivery status of each notification that you send. Here is some [documentation](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/va-notify#delivery-status-callbacks) regarding the callbacks that VA Notify provides (not `vets-api` specific).
+If you need any further clarification or help during the integration process, feel free to reach out:
 
-Please note, currently callbacks are limited to 1 per service. If you are in a shared service or need different callback webhooks by use case please contact VA Notify. We are actively working on an enhancement to support this!
+- Slack Channel: #va-notify-public
```

Comment:
Can you make this a link?

Reply:
Fixed. I tried different variations of linkage.